tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
I think the problem is that you didn't explicitly configure any logging handlers and formatters... check out the code I linked at the beginning, it even comes with colorful output!